wolfcrypt/src/ecc.c

```diff
@@ -4854,6 +4865,11 @@ static int wc_ecc_sign_hash_hw(const byte* in, word32 inlen,
                return BAD_COND_E;
             }
         }
+    #elif defined(WOLFSSL_SILABS_SE_ACCEL)
+        err = silabs_ecc_sign_hash(in, inlen, out, outlen, key);
+        if (err != 0) {
+               return BAD_COND_E;
```

Comment:
You might consider using `WC_HW_E` as the error code?

Reply:
I was following the pattern from a few lines above from `PLUTON_CRYPT_ECC` section.